### DISCUSSION TITLE
python升级到3.9以上后，chia居然不能用了

### DISCUSSION THREAD

**Discussion by arde2020** [Chia-Network/chia-blockchain#6400] [Support]
是否需要降级到3.8，还是官方有啥解决方法？

**Comment by XIII39** [comment]
My installs run the Chia blockchain on Ubuntu 20.04 with Python 3.8 (not 3.9), no problem.
Are you on Ubuntu 21.04?
What is the error you're getting with 3.9?

**Reply by arde2020** [reply]
yes,on Ubuntu 21.04

**Comment by cccat6** [comment]
如果你用的是Linux，那么你需要重新编译安装。因为你按照官方步骤安装的话是从源码编译，你编译的版本是3.8，而后单独升级到3.9，是有可能不能用的。
如果是Windows，那不可能有影响。Windows的客户端都是预编译的通用可执行程序。

**Comment by BrandtH22** [comment]
@arde2020 , if you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
